projects/js-toolkit/package.json

```diff
@@ -3,12 +3,9 @@
 	"private": true,
 	"scripts": {
 		"build": "node scripts/build.js && node scripts/copyfiles.js --all",
-		"check-deps": "node scripts/check-deps.js",
-		"ci": "yarn clean && yarn check-deps && yarn build && yarn test",
```

Comment:
You probably shouldn't entirely delete the `ci` job; we have that in all the other `package.json` so you can run it from a project (or package) directory for convenience.

- Example project-level: [npm-scripts](https://github.com/liferay/liferay-frontend-projects/blob/e24eae81b7fb14180538ad22589ae245b5e9933c/projects/npm-tools/package.json#L14).
- Example package-level: [npm-bundler](https://github.com/liferay/liferay-frontend-projects/blob/e24eae81b7fb14180538ad22589ae245b5e9933c/projects/js-toolkit/packages/npm-bundler/package.json#L34).

Reply:
Done.

![done](http://www.quickmeme.com/img/51/5182a356723cc8b543e70c9ac7ca293438a02452b01e02061712bd568482b236.jpg)